src/pkg/selectors/sharepoint.go

```diff
@@ -315,6 +353,8 @@ const (
 	SharePointListItem    sharePointCategory = "SharePointListItem"
 	SharePointLibrary     sharePointCategory = "SharePointLibrary"
 	SharePointLibraryItem sharePointCategory = "SharePointLibraryItem"
+	SharePointPage        sharePointCategory = "SharePointPage"
+	SharePointPageItem    sharePointCategory = "SharePointPageItem"
```

Comment:
We did a naming inversion with libraries in lists, because the "thing" itself (a List, or a Library) is a container.  Therefore we have, for example, Libraries and LibraryItems.

I think that pages, however, are more suited to the pattern we see elsewhere, like in exchange.  Most cases, the "thing" is the item.  Eg: Mail, or Contacts.  So we suffix "folder" (or whatever container) onto the thing.

Having a `PageFolder` that holds `Pages` as items seems to fit paradigm more closely.  Mostly because I'd expect someone who says, "i want to restore a page" to be referencing the file itself, and not the directory.

```suggestion
	SharePointPageFolder        sharePointCategory = "SharePointPageFolder"
	SharePointPage    sharePointCategory = "SharePointPage"
```

Reply:
Resolved within [82b5d21](https://github.com/alcionai/corso/pull/2111/commits/82b5d21f190583a634c1fc0fabeadcf50aabbe11)